packages/peregrine/lib/talons/WishlistPage/useActionMenu.js

```diff
@@ -44,6 +47,22 @@ export const useActionMenu = (props = {}) => {
         { error: updateWishlistErrors, loading: isEditInProgress }
     ] = useMutation(updateWishlistMutation);
 
+    const { data: storeConfigData } = useQuery(
+        operations.getWishlistConfigQuery,
+        {
+            fetchPolicy: 'cache-and-network',
+            nextFetchPolicy: 'cache-first'
+        }
+    );
+
+    const shouldRender = useMemo(() => {
+        return (
+            storeConfigData
```

Comment:
Should maybe check for `storeConfigData.storeConfig` as well just to avoid possible null pointers.

You can also probably check storeConfig.enable_multiple_wishlists without checking for the hasOwnProperty. Since, if it doesn't, it will be undefined, so !== '1'

Reply:
I just put the whole thing in a try catch block and thus any possible null pointer would just return false.